libs/ardour/plugin.cc

```diff
@@ -592,5 +592,5 @@ PluginInfo::is_utility () const
 bool
 PluginInfo::is_analyzer () const
 {
-	return (category == "Analyser" || category == "Anaylsis" || category == "Analyzer");
+	return (category == "Analyser" || category == "Analysis" || category == "Analyzer");
```

Comment:
Bug fix?

Reply:
Maybe. LV2 and VST3 uses URIs/IDs to identify categories, this way there is no localized text needed.
Perhaps we should make those strings translatable, but I don't know how VST2 handles this.

Reply:
Pleease advise on how to proceed?